### PR TITLE
Makes shiny versions of lavaland monsters slightly more distinct

### PR DESCRIPTION
**Pull request by ChangelingRain**
- Magmawing watchers have more magma-y wings, will glow, and the projectile they fire will set you on fire for one life tick. They have a high chance to drop a unique crusher trophy if killed by a crusher.
- Icewing watchers will fire at the normal rate, but the hits from their blasts will freeze you in place for 0.8 seconds. They have an increased chance to drop a crusher trophy if killed by a crusher.
- Ancient goliaths will spawn random tentacles around them constantly, have more health, and move slower. They have an increased chance to drop a crusher trophy if killed by a crusher.
- Dwarf legions will actually drop dwarf corpses. They have a slightly increased chance to drop a crusher trophy if killed by a crusher.

### CODE REVIEW

**Comment by DoctoreMad** [comment]
The 1% boost was fine, really.

**Comment by ChangelingRain** [comment]
About one of these spawns per round, and there's no signal to find it. 1% is not very much.

**Comment by Xhuis** [comment]
I'm fine with the sprites, but the rest doesn't strike me as necessary and I feel it should be cut. They haven't even had time to be on the live servers yet.

**Comment by DoctoreMad** [comment]
@ChangelingRain It doesn't have to be "very much". Mining really doesn't need more loot bags walking around.

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm [hunk 1]
```diff
@@ -87,62 +86,87 @@
...
 	pre_attack_icon = "Goliath_preattack"
 	throw_message = "does nothing to the rocky hide of the"
 	loot = list(/obj/item/stack/sheet/animalhide/goliath_hide) //A throwback to the asteroid days
+	butcher_results = list(/obj/item/reagent_containers/food/snacks/meat/slab/goliath = 2, /obj/item/stack/sheet/bone = 2)
+	crusher_drop_mod = 75
+
+/mob/living/simple_animal/hostile/asteroid/goliath/beast/ancient/Life()
+	. = ..()
+	if(isturf(loc))
+		for(var/turf/open/T in orange(4, loc))
```
Comment (RemieRichards) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm]: >```orange()``` every life tick

cache it, and the turf it was taken from (eg: the loc at the time), and only update it:
- every 4 life ticks
or
- the goliath moves a set distance from the turf the original was taken from.

Comment (optimumtact) [reply code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm]: you can't escape the orange

**Comment by Repukan** [comment]
Don't make it guaranteed drop chance.

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm [hunk 2]
```diff
@@ -87,62 +86,100 @@
...
+			new /obj/effect/temp_visual/goliath_tentacle(T, spawner)
 
-/obj/effect/goliath_tentacle/proc/Trip()
-	for(var/mob/living/M in src.loc)
-		visible_message("<span class='danger'>The [src.name] grabs hold of [M.name]!</span>")
-		M.Stun(100)
-		M.adjustBruteLoss(rand(10,15))
+/obj/effect/temp_visual/goliath_tentacle/proc/trip()
+	icon_state = "Goliath_tentacle_wiggle"
+	sleep(3)
```
Comment (Cyberboss) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm]: Timer

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**